packages/amplify-storage-simulator/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "amplify-storage-simulator",
+  "version": "0.0.1",
+  "description": "An S3 simulator to test S3 APIs",
+  "main": "lib/index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "tsc",
+    "watch": "tsc -w",
+    "start": "node lib/test/test.js"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@hapi/topo": "^3.1.2",
+    "@types/body-parser": "^1.17.0",
```

Comment:
All the `@types/*` packages should be devDependencies

Reply:
moved all required @ types to dev dependencies